packages/react-data-grid/src/Cell.js

```diff
@@ -297,6 +297,18 @@ const Cell = React.createClass({
     }
   },
 
+  removeScroll() {
+    let ctrl: any = this; // flow on windows has an outdated react declaration, once that gets updated, we can remove this
+    if (ctrl.isMounted()) {
```

Comment:
This is an anti pattern, any reason for using it?
https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html